### ISSUE TITLE
[Feature Request]: Add DOP Circle to map for connected node

### ISSUE THREAD

**Issue by b8b8** [meshtastic/Meshtastic-Android#1239]
### Tell us your idea.

I think it would be a good idea to add the Horizontal DOP to the android app map as just a simple thin black circle. This is already something that can be included in the position packets, so it makes sense to show it. It can also be shown for all other nodes though I think in that case it will make the map too hectic since other nodes already have their precision circles shown. We are getting some nice and robust GPS code recently along with many hardware GPS now supporting multiple constellations and multiple frequencies so it would be beneficial to show it.

https://en.wikipedia.org/wiki/Dilution_of_precision_(navigation)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jamesarich** [comment]
@b8b8 has this been resolved by #1177 ?

**Comment by jamesarich** [comment]
oh - rereading this is in addition to precision circles.

**Comment by b8b8** [comment]
> oh - rereading this is in addition to precision circles.

Correct, it's for a single thin lined GPS DOP circle (that can be toggled on and off)

**Comment by garthvh** [comment]
This data is pretty hard to interpret, it is not an accuracy in meters like the os positions provide. I have never been able to use it for anything.

**Comment by b8b8** [comment]
> This data is pretty hard to interpret, it is not an accuracy in meters like the os positions provide. I have never been able to use it for anything.
> 
> The computation section on Wikipedia demonstrates all the necessary computations to make it useful.

Can i ask then why both DOP and HVDOP are selectable to send with a position packet?

**Comment by garthvh** [comment]
It was added while the project did not really have a leadof by a gps interested contributor.

**Comment by garthvh** [comment]
https://github.com/meshtastic/firmware/issues/3984

**Comment by jamesarich** [comment]
closing this, doesn't sound like we can use this as originally proposed.
